---
title: "Integrating hyperspectral and thermal images in genetic studies: a case of study in maize under inoculation with plant growth-promoting bacteria"
author:  
 - Rafael Massahiro Yassue
  
institute: "ESALQ/USP and Virginia Tech"
date: "`r Sys.Date()`"
output:
  xaringan::moon_reader:
    lib_dir: libs
    css:
      - default
      - default-fonts 
      - animate.css

---
class: animated, fadeIn

# Summary
+ Updates

--

+ Motivations
 - Why study plant growth-promoting bacteria (PGPB)?
 - Why study the genetic architecture of the response to PGPB?
 - Why use hyperspectral imagery?
 - Why use thermal imagery?

--

+ Material and Methods

--

+ Studies:
 1. Classification of plant growth-promoting bacteria inoculation status and prediction of growth-related traits in tropical maize using hyperspectral image and genomic data
 2. Integrating hyperspectral and multi-phenotype genome-wide association analysis to dissect the genetic architecture of growth-related phenotypes in maize under inoculation with plant growth-promoting bacteria 
 3. Assessing plant vigor in the early stage of maize development under inoculation with plant growth promoting bacteria through low-resolution thermal images
 
---
class: animated, fadeIn
```{r, echo=FALSE, out.width="120%",fig.cap="", fig.align="center"}
knitr::include_graphics("TPPJ.png")
```

---
class: animated, fadeIn

# Why study plant growth-promoting bacteria (PGPB)?
```{r, echo=FALSE, out.width="95%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/stronger_plants.png")
```

---
class: animated, fadeIn

## Why study the genetic architecture of the response to PGPB?
```{r, echo=FALSE, out.width="95%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/figures.png")
```

---
class: animated, fadeIn
# Why use hyperspectral imagery?
```{r, echo=FALSE, out.width="95%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/why_Hyperpsectral.png")
```

---
class: animated, fadeIn
# Why use Thermal imagery?
```{r, echo=FALSE, out.width="95%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/Why thermal.png")
```

---
class: animated, fadeIn

# Goal 

+ Using hyperspectral and thermal image and genomic data to assess the genetic architecture of the symbioses between maize and plant growth-promoting bacteria and improve the predictive ability for growth-related traits 

<br /> <br /> <br /> <br />

```{r, echo=FALSE, out.width="150%"}
knitr::include_graphics("Figures/figs1.png")
```

---
class: animated, fadeIn

# Materials used throughout the studies 
+ Greenhouse condition
+ Early stage of maize development (six fully expanded)
+ 360 inbred lines
+ 13,826 SNPs
+ Two management both under nitrogen limited conditions: 
  - **B+** Syntetic population of 4 PGPB: 
      * *Bacillus thuringiensis* RZ2MS9
      * *Delftia* sp. RZ4MS18 
      * *Pantoea agglomerans* 33.1
      * *Azospirillum brasilense* Ab-v5
  - **B-** Without PGPB 
+ Manually measured phenotypes: 
  - Plant height (PH)
  - Stalk diameter (SD) 
  - Shoot dry mass (SDM)
+ Hyperspectral phenotypes: 
  - 150 wavelengths (from 386 to 1021 nm)
  - 131 hyperspectral index
+ Thermal phenotypes


---
class: animated, fadeIn
# Hyperspectral image acquisition
```{r, echo=FALSE, out.width="90%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/fig01_da.png")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Summary of data acquisition and processing. A) the hyperspectral benchtop camera used for data collection; B) region of interest of the last completed expanded leaf; C) cube image; and D) image mask
</p>

---
class: animated, fadeIn
# Exploratory analysis
```{r, echo=FALSE, out.width="70%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/2-1.jpeg")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. A: Spectral signatures of the maize genotypes with (B+) and without (B-) PGPB inoculation. B: Principal component biplot of the 360 maize genotypes and 150 spectral wavelengths with (B+) and without (B-) PGPB inoculation.
</p>

---
class: animated, fadeIn
# Exploratory analysis
```{r, echo=FALSE, out.width="70%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/2-2.jpeg")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Phenotypic correlations between plant height (PH), Stalk diameter (SD), and shoot dry mass (SDM) and hyperspectral reflectance (left) and hyperspectral index (right) with and without PGPB inoculation (B+ and B-).
</p>

---
class: inverse, animated, fadeIn
 <br /> <br /> <br /> <br /> <br /> <br />
<center>
<h1>
Classification of plant growth-promoting bacteria inoculation status and prediction of growth-related traits in tropical maize using hyperspectral image and genomic data
</h1>
</center>

---
class: animated, fadeIn
# Goals

+ Evaluate the utility of hyperspectral image data to classify PGPB inoculation status

+ Compare the predictive ability of genomic prediction, hyperspectral prediction, and their combination for growth-related phenotypes under PGPB management.  

---
class: animated, fadeIn
# Cross-validation scheme
```{r, echo=FALSE, out.width="80%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/1-1.jpeg")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Graphic representation of the cross-validation design using the hyperspectral image and genomic data for A. classification of inoculation status and B. Prediction of growth-related phenotypes.
</p>
---
class: animated, fadeIn
# Single and multi-omic models
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Table. List of models and covariates included in the analysis.
</p>
```{r, echo=FALSE, out.width="70%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/1-2.jpeg")
```
---
class: animated, fadeIn
# Classification and prediction models
* AutoML: Automated machine learning (Auto-Sklearn)
```{r, echo=FALSE, out.width="90%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/Auto-sklearn.png")
```

--

* Single-omic prediction
`\begin{align*}
\mathbf{y}=\mathbf{X b}+\sum_{i=1}^{\mathrm{m}_{\mathrm{HSI}}} \mathbf{W}_{\mathrm{HSI}_{i}} a_{\mathrm{HSI}_{i}}+\mathbf{e}
\end{align*}` 

`\begin{align*}
\mathbf{y}=\mathbf{X b}+\sum_{i=1}^{\mathrm{m}_{\mathrm{SNP}}} \mathbf{W}_{\mathrm{SNP}_{i}} a_{\mathrm{SNP}_{i}}+\mathbf{e}
\end{align*}` 

--

* Multi-omic models (Multiclass Bayesian analysis)
`\begin{align*}
\mathbf{y}=\mathbf{X} \mathbf{b}+\sum_{i=1}^{\mathrm{m}_{\mathrm{HSI}}} \mathbf{W}_{\mathrm{HSI}_{i}} a_{\mathrm{HSI}_{i}}+\sum_{j=1}^{\mathrm{m}_{\mathrm{SNP}}} \mathbf{W}_{\mathrm{SNP}_{j}} a_{\mathrm{SNP}_{j}}+\mathbf{e}
\end{align*}` 

---
class: animated, fadeIn

# PGPB inoculation classification 
```{r, echo=FALSE, out.width="60%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/1-3.png")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Classification accuracy of inoculation status (B+ and B-) using 150 hyperspectral bands. 
</p>

---
class: animated, fadeIn
# Growth-related phenotypes prediction
```{r, echo=FALSE, out.width="80%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/1-4.png")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Predictive ability of plant height (PH), stalk diameter (SD), and shoot dry mass (SDM) using phenomic prediction, genomic prediction, and multi-omic prediction models in each management condition (B+ and B-). 
</p>


---
class: animated, fadeIn
# Conclusion

+ Hyperspectral reflectance data can classify plant growth-promoting bacteria inoculation status

+ Phenomic prediction performs better than genomic prediction depending on the target phenotype

+ AutoML is a promising approach for automating hyperparameter tuning for classification and prediction

---
class: inverse, animated, fadeIn
 <br /> <br /> <br /> <br /> <br />
<center>
<h1>
Integrating hyperspectral and multi-phenotype genome-wide association analysis to dissect the genetic architecture of growth-related phenotypes in maize under inoculation with plant growth-promoting bacteria
</h1>
</center>

---
class: animated, fadeIn
# Goals

+ Develop a pipeline to integrate phenomics data into GWAS framework

+ Employ multi-phenotype GWAS, PheWAS, and gene annotation  in order to identify possible candidate genes to pleiotropic effects 

+ Identify candidate phenotypes responsible for changes in the hyperspectral reflectance

---
class: animated, fadeIn
# Materials and Methods
+ Phenotypes:
  - Manually measured phenotypes: 
     * Plant height (PH)
     * Stalk diameter (SD) 
     * Shoot dry mass (SDM)
  - Hyperspectral phenotypes: 
     * 150 wavelengths (from 386 to 1021 nm)
     * 131 hyperspectral index

+ Bayesian GWAS
  - BayesC 
  - $\pi$ = 0.99
  - Model frequency 

+ Phenome-wide association studies (PheWAS)

---
class: animated, fadeIn
# Single trait BayesC
In order to estimate the markers effect and  variance components, the following model were used for each combination of phenotype and management (B+ and B-):

` \begin{align*}
\boldsymbol{y}_{rli}=\boldsymbol{\mu}+\boldsymbol{r}_k+\boldsymbol{b}_l+ \sum_{j=1}^{p} m_{i j} \boldsymbol{\alpha}_{j}+\boldsymbol{e}_{rli}
\end{align*}`

`\begin{align*}
\alpha_{j} | \pi, \sigma^{2}_{j}  = \begin{cases}
0 & \text{ with probability of } \pi \\
\sim N (0, \sigma^{2}_{j}) & \text{ with probability } (1 - \pi)
\end{cases}
\end{align*}` 

and $\pi$ = 0.99.

---
class: animated, fadeIn
# Bivariate BayesC

` \begin{align*}
\boldsymbol{y}_{rli}=\boldsymbol{\mu}+\boldsymbol{r}_k+\boldsymbol{b}_l+ \sum_{j=1}^{p} m_{i j} \boldsymbol{\alpha}_{j}+\boldsymbol{e}_{rli}
\end{align*}`

` \begin{align*}
\boldsymbol{\alpha}_{j}\ =\boldsymbol{D}_{j}\boldsymbol{\beta}_{j}
\end{align*}`

` \begin{align*}
\operatorname{\textit{diag}}\left(\boldsymbol{D}_{j}\right)=\delta_{j}=\left(\delta_{j 1}, \delta_{j 2}, \delta_{j 3} \ldots \delta_{j t}\right)
\end{align*}`

` \begin{align*}
p\left(\delta_{l}="i"\right)=\Pi_{1} I\left(\delta_{l}=" 1 "\right)+\Pi_{2} I\left(\delta_{l}=" 2^{\prime \prime}\right) + ... + \Pi_{4} I\left(\delta_{l}=" 4^{\prime \prime}\right)
\end{align*}`



` \begin{align*}
\boldsymbol{\beta}_{j} \sim MVN (Null, \mathbf{G}) 
\end{align*}`


where 
` \begin{align*}
\mathbf{G} = \begin{bmatrix} \sigma^{2}_{g_1}  & \sigma_{g_{12}} \\  \sigma_{g_{21}} & \sigma^{2}_{g_{2}}  \end{bmatrix}, \hspace{0.2cm}
\end{align*}`


---
class: animated, fadeIn
# Genomic heritability
```{r, echo=FALSE, out.width="70%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/2-3.jpeg")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Genomic heritability for 150 wavelengths and 131 hyperspectral indices with and without inoculation with PGPB (B+ and B-)
</p>

---
class: animated, fadeIn
# Genomic correlation
```{r, echo=FALSE, out.width="70%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/2-4.jpeg")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Genomic correlation for 150 wavelengths and 131 hyperspectral indices with and without inoculation with PGPB (B+ and B-)
</p>

---
class: animated, fadeIn
#  Manually measured phenotype GWAS
```{r, echo=FALSE, out.width="70%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/2-5.jpeg")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Manhatan plot for manually measured phenotypes (plant height, PH; stalk diameter, SD; and shoot dry mass, SDM) under with and without PGPB inoculation (B+ and B-)
</p>

---
class: animated, fadeIn
# Phenome-wide association studies (PheWAS)
```{r, echo=FALSE, out.width="100%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/2-6.png")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. PheWAS for the 10 highest model frequency SNPs for plant height (PH), stalk diameter (SD), and shoot dry mass (SDM) for B+ management
</p>
---
class: animated, fadeIn
# Multi-phenotype GWAS
```{r, echo=FALSE, out.width="70%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/2.gif")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Multi-phenotype GWAS for the 281 hyperspectral phenotype under B+ management 
</p>

---
class: animated, fadeIn
# Multi-phenotype GWAS
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Table. Summary of multi-phenotype GWAS for the 281 hyperspectral phenotypes and gene annotation.
</p>
```{r, echo=FALSE, out.width="70%",fig.cap="", fig.align="center"}
knitr::include_graphics("papers/2-8.jpeg")
```


---
class: animated, fadeIn
# Shiny app
```{r, echo=FALSE, out.width="90%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/1.jpeg")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Shiny APP
</p>
management
---
class: animated, fadeIn
# Shiny app
```{r, echo=FALSE, out.width="90%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/2.jpeg")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Shiny APP
</p>

---
class: animated, fadeIn
# Conclusion

+ Hyperspectral reflectance can assess the genetic architecture of growth-related phenotypes in maize

+ PheWAS analysis is a promossing tool for genetic architecture studies using hyperspectral data

+ Shiny app for multi-phenotype Manhattan plots is a user-friendly tool

---
class: inverse, animated, fadeIn
 <br /> <br /> <br /> <br /> <br />
<center>
<h1>
Assessing plant vigor in the early stage of maize development under inoculation with plant growth-promoting bacteria through low-resolution thermal images
</h1>
</center>
---
class: animated, fadeIn

# Goal 

+ Use thermal image to assess plant stress

+ Validate thermal data using manually mesuared and hyperspectral image data

+ Understand the genetic architecture of thermal phenotypes under inoculation with PGPB

+ Provide a Python package for low-resolution thermal image processing for conveyor HTP platform

---
class: animated, fadeIn
# Thermal data collection

```{r  , echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default"}
knitr::include_graphics("Figures/RGB_colection.gif")
knitr::include_graphics("Figures/thermal_coletion.gif")
```

<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure.  RGB (left) and thermal (right) image collection using a low-cost greenhouse-based high-throughput phenotyping platform
</p>


---
class: animated, fadeIn
# Materials and Methods

+ Thermal image acquisition: 
    - Flir vue pro R: 336 × 256 pixels

--

+ Processing Low-resolution thermal images: 
   - Converting radiation to Celsius degree
   - Stitching group images 
   - Identify local coordinates for each plot in each image
   - Identify global coordinates for each pot in the whole experiment
   - Extract information:
      * Image segmentation using watershed algorithm

--

+ Assessing plant stress 
   - Canopy temperature 
   - Validate using manually measured phenotypes and hyperspectral data:
      * Phenotypic and genomic correlation
      * GWAS
      * Gene annotation

---
class: animated, fadeIn
# Image stitching and pot identification
```{r, echo=FALSE, out.width="150%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/Thermal_01.png")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Thermal images used for stitching (1-5). Representation of the stitched image. 
</p>
---
class: animated, fadeIn
# Local coordinates
```{r, echo=FALSE, out.width="70%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/thermal02.png")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Thermal image processing. Stitched image, plot detection, and local coordinates. 
</p>

---
class: animated, fadeIn
# Global coordinates
```{r, echo=FALSE, out.width="90%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/thermal_2.gif")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Representation of the global coordinate across several stitched images.
</p>
---
class: animated, fadeIn
#  Computing global coordinates
```{r, echo=FALSE, out.width="49%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/GC_global.png")
```

```{r, echo=FALSE, out.width="45%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/GC.gif")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Representation of global coordinate estimation.
</p>

---
class: animated, fadeIn
#  Extract information
```{r, echo=FALSE, out.width="85%",fig.cap="", fig.align="center"}
knitr::include_graphics("Figures/Extract_info.png")
```
<p style="font-family: times, serif; font-size:11pt; font-style:italic">
Figure. Schematic representation of the information extraction from the stitched thermal images.
</p>


---
class: animated, fadeIn
#  Expect results
+ Plants in B+ management has lower temperature than in the B-

+ Moderate correlation with hyperspectral phenotypes and manually measured

+ Some SNPs associated with canopy temperature, nitrogen uptake, and abiotic stress

+ Python package 

---
class: animated, fadeIn
# Next steps

+ Integrating high-throughput phenotyping and random regression to dissect the genetic architecture of plant growth-promoting bacteria response in tropical maize

---
class: animated, fadeIn

<br /> <br /> <br /> <br /> <br /> <br />
#.center[Thank you]

###.right[ Rafael Massahiro Yassue ]
.right[[rafael.yassue@usp.br](mailto:rafael.yassue@usp.br?subject=%20Presentation)]
.right[https://rafaelyassue.github.io/cv-online/]
```{r, echo=FALSE, out.width="50%",fig.cap="", fig.align="right"}
knitr::include_graphics("Figures/site.png")
```